scripts/localk8s/run-integration-tests.bat

```diff
@@ -0,0 +1,50 @@
+@ECHO OFF
+SET SCRIPTLOC=%~dp0
+
+:: This script will modify the Kafka topic configuration to use topics prefixed 'itg-'
+:: before running the tests, and then reset the topics when finished.
+
+:: Delete old integration job, if it exists
+kubectl delete -f %SCRIPTLOC%\..\..\itg-tests\es-it\ReeferItgTests.yaml -n shipping
+
+:: Deploy integration Kafka topics
+kubectl apply -f %SCRIPTLOC%\itg-kafka-topics-configmap.yaml -n shipping
+kubectl apply -f %SCRIPTLOC%\itg-topics.yaml -n kafka
+
```

Comment:
Need to set the `bpm-anomaly` url and login attributes to the BPM mockup endpoints: https://github.com/ibm-cloud-architecture/refarch-kc/pull/72